src/pages/ImportData.js

```diff
@@ -95,9 +95,9 @@ const ResponseFragment = ({ title, snapshotResponses, responseIndex }) => {
   ]);
 };
 
-const ImportDataOverview = ({ header, snapshots, isDataset, snapshotResponses, url }) =>
+const ImportDataOverview = ({ header, snapshots, isDataset, snapshotResponses, url, format }) =>
   div({ style: styles.card }, [
-    h2({ style: styles.title }, [header]),
+    h2({ style: styles.title, tabindex: 0 }, [header]),
```

Comment:
Why does the header need to be focusable?

Reply:
I was just struggling to navigate by keyboard and couldn't figure out how to get to it!  I can take this out if it's not needed.

Reply:
You should be able to navigate to the link (with Tab) without this… let me know if that's not the case and I can take a look (I don't know how to get to this page).

As far as making sure a screen reader is aware of the message, you could try adding a role of 'alert' to the message. This is especially important if it appears after the page is initially rendered (for instance, triggered by selecting something on the page).

Reply:
Tab was not taking me to that section of the page.  You get to that page from importing from a third party - such as going to [TDR](https://jade.datarepo-dev.broadinstitute.org/snapshots), selecting a snapshot and exporting it.  The message should be there when the page is initially rendered - should it still have an 'alert' role?

Reply:
Do you mean that you aren't able to Tab to the "Read more" link?

No, if the content is already there at the time the page is loaded you don't want to add an alert role to it.

Reply:
I can tab to the link but not the rest of the text - so I couldn't get the screen reader to read this important message!

Reply:
I got to it eventually, I guess I'm just not very comfortable with keyboard navigation and voiceover.  PEBKAC

Reply:
You don't need to be able to tab to all text… only controls that take input or do actions should be keyboard focusable.

Screen reader users read non-focusable text in other ways. Screen reader applications have ways to read the text of a page from top to bottom, and also ways to navigate non-focusable content for the purposes of reading the text.

We can try this out in Wednesday's course group meeting if you would like.

Reply:
Here is a reference for VoiceOver: https://www.apple.com/voiceover/info/guide/_1128.html

Reply:
Thank you!